### PR TITLE
Fix visibility of interfaces in \OCP

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @rullzer @nickvergessen self explanatory

### CODE REVIEW

File: lib/public/iavatarmanager.php [hunk 1]
```diff
@@ -38,5 +38,5 @@
 	 * @return \OCP\IAvatar
 	 * @since 6.0.0
 	 */
-	function getAvatar($user);
+	public function getAvatar($user);
```
Comment (Xenopathic) [inline lib/public/iavatarmanager.php]: Fix corresponding `lib/private/avatarmanager.php` line?

File: lib/public/icontainer.php [hunk 2]
```diff
@@ -57,7 +57,7 @@ function query($name);
 	 * @return void
 	 * @since 6.0.0
 	 */
-	function registerParameter($name, $value);
+	public function registerParameter($name, $value);
```
Comment (Xenopathic) [inline lib/public/icontainer.php]: Needs fix in `lib/private/appframework/utility/simplecontainer.php`

File: lib/public/icontainer.php [hunk 3]
```diff
@@ -72,5 +72,5 @@ function registerParameter($name, $value);
 	 * @return void
 	 * @since 6.0.0
 	 */
-	function registerService($name, \Closure $closure, $shared = true);
+	public function registerService($name, \Closure $closure, $shared = true);
```
Comment (Xenopathic) [inline lib/public/icontainer.php]: Needs fix in `lib/private/appframework/utility/simplecontainer.php`

File: lib/public/irequest.php [hunk 4]
```diff
@@ -131,7 +131,7 @@ public function getEnv($key);
 	 * @return array the value in the $_COOKIE element
 	 * @since 6.0.0
 	 */
-	function getCookie($key);
+	public function getCookie($key);
```
Comment (Xenopathic) [inline lib/public/irequest.php]: Needs fix in `lib/private/appframework/http/request.php`

**Comment by nickvergessen** [comment]
:+1: when the comments are fixed

**Comment by rullzer** [comment]
Same for me. :+1: once the comments are fixed

**Comment by nickvergessen** [comment]
Fixed the comments

**Comment by MorrisJobke** [comment]
:+1: for the fixes

**Comment by Xenopathic** [comment]
:+1: @DeepDiver1975 merge at your discretion :rocket:

**Pull request merged**